src/tpl/index.html

```diff
@@ -88,5 +88,8 @@ <h3 class="feature-title">{{title}}</h3>
     </ul>
   </article>
   <script src="bundle.js" async defer></script>
+  <footer>
+    Found an issue with this page? <a href='https://github.com/mozilla/platatus/issues/new'>File an issue!</a>
```

Comment:
A lot of *issues*; we could make it more inviting and positive.

Taking a shot at wordsmithing: 2 links, "Fork on Github" and "Submit Feedback"